### PR TITLE
Add menu theming

### PR DESCRIPTION
**Pull request by PROGRAM-IX**
Second of three PRs based on svexican/cheeky#4, breaking it up into more granular chunks.

- [x] Add `menu_theme` option to change the switcher's appearance based on awesomewm menu theme options
- [x] Explain use of `menu_theme` option in README

### CODE REVIEW

**Comment by svexican** [comment]
The pull request **at this point in history** has conflicts. It didn't when you first created the PR. See comments. Would you like me to commit this or wait for the new PR? (It would appear as _my_ commit, and I'm not a fan of that sort of mishappenings...)

File: README.md [hunk 1]
```diff
@@ -67,9 +67,14 @@ The full table you can pass to the `switcher` function (some are pretty useless
     hide_notification = false,   -- default: true
     notification_text = "NOPE",  -- default: "No matches. Resetting"
     notification_timeout = 5     -- default: 1
```
Comment (svexican) [inline README.md]: This line now looks like: `... timeout = 5,     -- default: 1... ` (notice the comma). But the PR assumes it doesn't have the comma.

**Comment by PROGRAM-IX** [comment]
I'll fix it. Sorry. :)

**Comment by svexican** [comment]
Check that it doesn't happen to the other PR. _history-wise_

**Comment by PROGRAM-IX** [comment]
Hm, I was afraid of this. I'll change the other PR as well. Don't worry, I'm not going to just create these and abandon them for you to fix.

**Comment by PROGRAM-IX** [comment]
Hmm, still have a conflict. I'll check it again.

**Pull request merged**